packages/template-retail-react-app/app/components/experience/image-tile/index.jsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import PropTypes from 'prop-types'
+import {Image} from '@chakra-ui/react'
+
+/**
+ * Simple Image tile component that can be used inside any Layout component
+ * @param image Object containing the image url, _type and focalPoint.
+ * @param rest The rest of the potential image parameters.
+ * @returns {JSX.Element}
+ */
+const ImageTile = ({image, ...rest}) => {
+    return (
+        <figure className={'image-tile-figure'}>
```

Comment:
From the looks of the components in SFRA they have a pattern where the outermost element of the component is labelled as "<component-name>-container". Although I'm not saying we need to duplicate this 1-1 I do think that the outermost element should be something other than a "figure" element. 

A div with a class name of `image-tile` would probably suffice.